web-components/src/merch-card.js

```diff
@@ -612,6 +612,8 @@ export class MerchCard extends LitElement {
                 !e.shiftKey &&
                 document.activeElement === lastFocusableElement
             ) {
+                const isLastMerchCard = this === document.querySelectorAll(MERCH_CARD_NODE_NAME).item(document.querySelectorAll(MERCH_CARD_NODE_NAME).length - 1);
```

Comment:
- do you have lint enabled on your vs code? this line looks too long :) 
- this looks suboptimal and non functional : PR title says "within a segment blade" but you are searching over the whole document. Can you have a look at a page test with different containers (like 2 segment blades) and see how it works? also you should be doing querySelectorAll only once.

Reply:
I think we have to do it for the last card only, we can navigate between the card segments using up and down arrows but it is the last card where the user can stuck.
Other points, I have taken care.